tests/Doctrine/DBAL/Migrations/Tests/Functional/FunctionalTest.php

```diff
@@ -13,6 +16,7 @@
 use Doctrine\DBAL\Migrations\Tests\Stub\Functional\MigrationModifySchemaInPreAndPost;
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use \Mockery as m;
```

Comment:
Is it widely used? While I understand that each project has its own preferences, I'd avoid having another mocking framework in here. I can replace the mocks myself, given some time